packages/gatsby-theme-docs/src/components/code-example.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { CodeBlock, ContentNotifications } from '@commercetools-docs/ui-kit';
+import useCodeExamples from '../hooks/use-code-examples';
+
+function CodeExample(props) {
+  const codeExamples = useCodeExamples();
```

Comment:
Wondering if it makes more sense to have the hook `useCodeExample` (singular), which returns a single code example:

```js
const codeExample = useCodeExample(props.path)
```

The advantage with this is that the logic to find the code example is encapsulated in the hook (less repetition in case it's used somewhere else).
Just something to consider...

Reply:
I first thought that's good too but then remembered that during the implementation of the API types hook which is the same code design I found that I need use case specific handling e.g. when multiple entries matched or when none matched.  So It's not as obvious and for now I would leave the current state.